src/components/widgets/ButtonSimple.vue

```diff
@@ -45,6 +45,12 @@
     <codepen-icon class="icon" v-else-if="icon == 'codepen'" />
     <link-icon class="icon" v-else-if="icon == 'link'" />
     <clock-icon class="icon" v-else-if="icon == 'clock'" />
+    <file-digit
```

Comment:
Use `file-digit-icon` to be consistent with other icon imports.
It is helpful to quickly find all the `*-icons` components.

```suggestion
    <file-digit-icon
```